runtime/src/runtime/policy.rs

```diff
@@ -428,6 +454,36 @@ impl ProofSet {
             proofs[i64::from(RegisteredSealProof::StackedDRG64GiBV1P1) as usize] = true;
             proofs[i64::from(RegisteredSealProof::StackedDRG64GiBV1P1_Feat_SyntheticPoRep)
                 as usize] = true;
+            proofs[i64::from(RegisteredSealProof::StackedDRG64GiBV1P2_Feat_NiPoRep) as usize] =
```

Comment:
This is used to validate pre-commits, so the NI-porep type shouldn't be here.

Could you rename the method to `default_precommit_proofs`?